### DISCUSSION TITLE
securityonion installation method

### DISCUSSION THREAD

**Discussion by mice-love-rice** [Security-Onion-Solutions/securityonion#9981] [Unsupported Versions]
Hi dear, I have encountered a problem. The problem is this. I remember that there is a way to install security onion before, which is to use docker directly without using the iso file, but I have been looking for it on the Internet for a long time. This method seems to have disappeared until I was in China. I saw the record URL on the website here https://juejin.cn/s/security%20onion%20docker%20configuration  Is this method not in use now?

**Comment by tomjones1977** [comment]
The only official installation methods are in the docs:

https://docs.securityonion.net/en/2.3/installation.html
